libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -87,7 +86,7 @@ class BitcoinLikeExplorer implements IExplorer {
     client.interceptors.response.use(responseInterceptor, errorInterceptor);
   }
 
-  async broadcast(tx: string): Promise<any> {
+  async broadcast(tx: string): Promise<AxiosResponse<any, any>> {
     const url = "/transactions/send";
     const client = await this.client.acquire();
     const res = await client.client.post(url, { tx });
```

Comment:
Discussion not related to the PR but we catch HTTP Errors

Reply:
Done here: https://github.com/LedgerHQ/ledger-live/blob/e89044242d005bfc3349abbbf1921e9056686d0b/libs/ledger-live-common/src/families/bitcoin/wallet-btc/explorer/index.ts#L86-L87
Bitcoin implem is using the same axios interceptors are our standard network util.